aiplatform/api/AIPlatform.Samples/PredictChatPromptSample.cs

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START aiplatform_sdk_chat]
+
+using Google.Cloud.AIPlatform.V1;
+using wkt = Google.Protobuf.WellKnownTypes;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+public class PredictChatPromptSample
+{
+    public string PredictChatPrompt(
+        string projectId = "your-project-id",
+        string locationId = "us-central1",
+        string publisher = "google",
+        string model = "chat-bison@001"
+    )
+    {
+        // Initialize client that will be used to send requests.
+        // This client only needs to be created once,
+        // and can be reused for multiple requests.
+        var client = new PredictionServiceClientBuilder
+        {
+            Endpoint = $"{locationId}-aiplatform.googleapis.com"
+        }.Build();
+
+        // Configure the parent resource.
+        var endpoint = EndpointName.FromProjectLocationPublisherModel(projectId, locationId, publisher, model);
+
+        // Initialize request argument(s).
+        var prompt = "How many planets are there in the solar system?";
+
+        // You can construct Protobuf from JSON.
+        var instanceJson = JsonConvert.SerializeObject(new
+        {
+            context = "My name is Miles. You are an astronomer, knowledgeable about the solar system.",
+            examples = new[]
+            {
+                new
+                {
+                    input = new { content = "How many moons does Mars have?" },
+                    output = new { content = "The planet Mars has two moons, Phobos and Deimos." }
+                }
+            },
+            messages = new[]
+            {
+                new
+                {
+                    author = "user",
+                    content = prompt
+                }
+            }
+        });
+        // var instance = wkt::Value.Parser.ParseJson(instanceJson);
+
+        // Or, you can construct Protobuf directly.
+        var instance = new wkt::Value
```

Comment:
My preference would be to show just one approach here to construct Protobuf directly. 
We can add a TODO stating, this would be greatly simplified when the new version of library is available and link to the documentation showing a possible second approach using JSON.

Reply:
Ack. I still think showing JSON in one of the samples would be useful, since Protobuf is so hard to work with. However, I'll let Amanda make the final call.

Reply:
@meteatamel Can you pick one sample to show JSON, have the rest with protobuf, and then explain in docs something like "This can also be done using JSON, please refer to sample xyz who demonstrates this"?

I find that the samples showing both are very long and for an unfamiliar user could end up being confusing.

I wouldn't mention a new version of the library as we don't have any ETA on that.

Reply:
Ok, I chose the JSON one in this sample and Protobuf in the other samples, so we can show both